packages/core/src/utils/index.ts

```diff
@@ -72,5 +72,14 @@ export function getElementFromInstance(
   return (instance?.vnode as VNode<DOMNode, DOMElement> | undefined)?.el
 }
 
+export function pickBy<T extends object>(
```

Comment:
let's remove this function, it's also quite expensive due to the different Object.* calls